sphinx/writers/latex.py

```diff
@@ -446,6 +446,18 @@ def __init__(self, document, builder):
             self.elements['multilingual'] = '%s\n\\setmainlanguage{%s}' % \
                 (self.elements['polyglossia'], self.babel.get_language())
 
+        # detect Chinese language, disable babel/polyglossia
```

Comment:
settings of babel and polyglossia are already finished here.
If you'd like to disable them, you should move these block to above.